File: problem_1/multiples_of_3_5.cpp
/*

*/

#include <iostream>

int main()
{
	int total = 0;
	
	for(int i = 0; i < 1000; i = i + 1)
	{
		if(i % 3 == 0 || i % 5 == 0)
		{
			total += i;
		}
	}
	
	std::cout << total << std::endl;
	std::cin;
	
	return 0;
}

File: problem_1/multiples_of_3_5.cs
/*
 *  This will also work with MONO
*/

using System;

namespace Multiples35
{
	class MainProgram
	{
		static void Main(string[] args)
		{
			int total = 0;

			for ( int i = 0; i < 1000; i = i + 1 )
			{
				if (i % 3 == 0 || i % 5 == 0)
				{
					total += i;
				}
			}

			Console.WriteLine(total);
			Console.ReadLine();
		}
	}
}
